flume-hdfs-sink/src/main/java/com/wipro/ats/bdre/flume/sink/hdfs/HDFSTextSerializer.java

```diff
@@ -64,8 +64,8 @@ private Object getKey(Event e) {
     return new LongWritable(eventStamp);
   }
 
-  private Object getValue(Event e) {
-    return makeText(e);
+  private Object getValue(Event event) {
+    return makeText(event);
   }
 
   public static class Builder implements SequenceFileSerializer.Builder {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS2176)